data/gaTable.yaml

```diff
@@ -983,3 +983,23 @@
   projectContactEmail: 'mpavlick@usgs.gov'
   analyticsContact: 'Meredith Pavlick'
   analyticsContactEmail: 'mpavlick@usgs.gov'
+
+- longName: "VizLab: Hurricane Harvey's Water Footprint"
+  shortName: VizLabHarvey
+  accountName: Vizlab
+  viewID: '159018840'
+  accountId: '78530187'
+  webPropertyId: UA-78530187-7
+  internalWebPropertyId: '157530047'
+  viewName: All Web Site Data
+  timezone: America/Chicago
```

Comment:
what do we do with this timezone field? looks like they're all America/Chicago?

Reply:
I think this gets used for the aggregated time of day plots.  There are a few (NWIS I think) that are other tzs.

Reply:
OK, thanks. i guess CDT makes sense for Harvey/Houston regardless